.travis.yml

```diff
@@ -15,16 +15,18 @@ php:
   - hhvm-nightly
 
 matrix:
-    allow_failures:
-        - php: nightly
-        - php: hhvm-nightly
-    fast_finish: true
+  include:
+    - php: 5.4
+      env: COMPOSER_FLAGS="--prefer-lowest"
+  allow_failures:
+    - php: nightly
+    - php: hhvm-nightly
+  fast_finish: true
 
 before_script:
-    - composer self-update
-
-install:
-  - composer install --dev
+  - composer self-update
+  - composer config -g github-oauth.github.com $GITHUB_OAUTH_TOKEN
```

Comment:
Can you remove this line for now? I'll take a deeper look to this feature but as we never reached GitHub quota until now, I don't think it's a priority.